CodingClubSource/GameEngineTests/Vector2Tests.cpp

```diff
@@ -100,5 +100,18 @@ namespace GameEngineTests
 			Assert::AreEqual(expected.x, vectorA.x);
 			Assert::AreEqual(expected.y, vectorA.y);
 		}
+
+		TEST_METHOD(Vector2_Length_Func_Returns_Length_Of_3_4_5_Triangle) // Right hand side to Left hand side
+		{
+			// Given
+			Vector2 testVector(3, 4);
+
+			// When
+			float actualLength = testVector.Length();
+
+			// Then
+			float expectedLength = 5.0f;
+			Assert::AreEqual(expectedLength, actualLength);
+		}
 	};
```

Comment:
All good. Might want a second test here for another length. Even if it's just for a similar triangle like

```
   /|
50/ |
 /  | 40
/   |
____
 30
```

But the function is pretty normal so probably all good